web/src/components/dashboard/SidebarCardFormationNudge.test.tsx

```diff
@@ -52,6 +52,7 @@ describe("<SidebarCardFormationNudge />", () => {
         preferences: generatePreferences({ visibleSidebarCards: [SIDEBAR_CARDS.formationNudge] }),
       });
 
+      expect(screen.getByTestId("formation-nudge")).toHaveClass("primary-gradient");
```

Comment:
pebble, generally speaking, I think each test should test 1 thing. If you're going to test the gradient, I recommend creating a new test. 

Separately, we tend to test logic. The classes applied to the cards are static Since this is the case, I'd lean on not having a test.

Reply:
deleted

Reply:
I think tests can be a little broader, but at the very least the expected functionality should be clearly described in the test name. In this case I'm in agreement, this could be covered in it's own test.

I don't think testing this functionality makes sense in this describe either. We're labeling this section `"when clicking formation button"`.